src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/ManagedIndexMetaData.kt

```diff
@@ -70,6 +76,31 @@ data class ManagedIndexMetaData(
         return resultMap
     }
 
+    fun toXContent(builder: XContentBuilder, params: ToXContent.Params, forIndex: Boolean): XContentBuilder {
```

Comment:
Possible to put forIndex into the ToXContent.Params?

Reply:
don't want to actually, would rather have 2 toXContent, since these 2 are very different. I think this way it's quite clear